spec/index.html

```diff
@@ -465,9 +452,7 @@ <h2>Canonicalization</h2>
     "Universal RDF Dataset Canonicalization Algorithm 2015"
     (<abbr title="Universal RDF Dataset Canonicalization Algorithm 2015"><dfn class="export">URDNA2015</dfn></abbr>).</p>
 
```

Comment:
```suggestion

    <div class="issue" data-number="87"></div>
    <div class="issue" data-number="88"></div>
```
Not hugely important, but as both #87 and #88 pertain to the name of the algorithm, I thought they would be better placed here.

Reply:
They’ll be removed in #116.